readme.txt

```diff
@@ -98,38 +98,20 @@ There are opportunities for developers at all levels to contribute. [Learn more
 
 = 7.8 =
 
-* Release date: October 1, 2019
-* Release post: https://wp.me/p1moTy-lvE
+* Release date: November 5, 2019

```

Comment:
We should also change the version two strings before this one: 7.9 instead of 7.8.